js/components/graph/Graph.js

```diff
@@ -625,6 +640,13 @@ export class Graph extends React.Component {
     })
   }
 
+  courseLinkMouseClick = courseCode => {
```

Comment:
No need for the word "mouse" here (this is true in general). In general when choosing names, look at existing functions and try to follow the same naming style.

In this specific case, however, I don't think the old `handleCourseClick` function is used any more, so you should actually just replace it with this function.